frontend/src/theme/components/DropdownMenu.ts

```diff
@@ -0,0 +1,76 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+
+export type DropdownMenuVariant = 'outline' | 'clear'
+export type DropdownMenuSize = 'normal' | 'stretch'
+
+export const DropdownMenu: ComponentMultiStyleConfig = {
+  parts: ['outerbox', 'innerbox', 'option'],
```

Comment:
the casing here is inconsistent (outerBox vs outerbox) - does the component still function as expected?

Reply:
apparently yes still - but I've fixed the casing